src/openrct2/ride/Ride.cpp

```diff
@@ -7333,10 +7320,7 @@ uint64_t Ride::GetAvailableModes() const
 
 const RideTypeDescriptor& Ride::GetRideTypeDescriptor() const
 {
-    if (type >= std::size(RideTypeDescriptors))
-        return DummyRTD;
-
-    return RideTypeDescriptors[type];
+    return ::GetRideTypeDescriptor(type);
 }
```

Comment:
This I the only function in this file that needs to use `::` the ride members can call this version and not need to pass the type